obs2ioda-v2/src/cxx/netcdf_utils.cc

```
@@ -0,0 +1,63 @@
+                std::remove_if(
+                        strippedName.begin(),
+                        strippedName.end(),
+                        [](unsigned char x) { return std::isspace(x); }
+                ),
+                strippedName.end()
+        );
+        return strippedName;
+    }
+
+
+    std::string getIodaName(
```

cppcheck (2.10-2)

[style] The function 'getIodaName' is never used.

(unusedFunction)